modules/planning/scenarios/park_and_go/stage_check.cc

```diff
@@ -55,6 +47,20 @@ Stage::StageStatus ParkAndGoStageCheck::FinishStage(const bool success) {
   return Stage::FINISHED;
 }
 
+void ADCInitStatus(Frame* frame) {
```

Comment:
If this is just a state shared among stages in this scenario, use ParkAndGoContext defined in the scenario class. 
PlanningContext is a static class for state shared among and beyond scenario/stage.